tiny_dnn/activations/softplus_layer.h

```diff
@@ -0,0 +1,49 @@
+/*
+    Copyright (c) 2017, Taiga Nomi
+    All rights reserved.
+
+    Use of this source code is governed by a BSD-style license that can be found
+    in the LICENSE file.
+*/
+#pragma once
+#include "tiny_dnn/activations/activation_layer.h"
+#include "tiny_dnn/layers/layer.h"
+
+namespace tiny_dnn {
+
+class softplus_layer : public activation_layer {
+ public:
+  using activation_layer::activation_layer;
+
+  std::string layer_type() const override { return "softplus-activation"; }
+
+  void forward_activation(const vec_t &x, vec_t &y) override {
+    for (serial_size_t j = 0; j < x.size(); j++) {
+      if (x[j] >= float_t(30)) {
```

Comment:
If `CNN_USE_DOUBLE` is defined, double precision floating point type is used for computation,
so using floating point literal `30.000000000000092` instead of `30` can be good to widen the line.
https://github.com/tiny-dnn/tiny-dnn/blob/0e372ddb1cf483e5d694a62c91868b6241c887ca/tiny_dnn/config.h#L88
http://www.exploringbinary.com/floating-point-converter/